man/sd_bus_message_get_type.xml

```diff
@@ -62,40 +74,51 @@
 
     <para><function>sd_bus_message_get_type()</function> returns the type of a message in the output
     parameter <parameter>type</parameter>, one of <constant>SD_BUS_MESSAGE_METHOD_CALL</constant>,
-    <constant>SD_BUS_MESSAGE_METHOD_RETURN</constant>,
-    <constant>SD_BUS_MESSAGE_METHOD_ERROR</constant>, <constant>SD_BUS_MESSAGE_SIGNAL</constant>.
-    This type is either specified as a parameter when the message is created using
+    <constant>SD_BUS_MESSAGE_METHOD_RETURN</constant>, <constant>SD_BUS_MESSAGE_METHOD_ERROR</constant>,
+    <constant>SD_BUS_MESSAGE_SIGNAL</constant>. This type is either specified as a parameter when the message
+    is created using
     <citerefentry><refentrytitle>sd_bus_set_message_new</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
     or is set automatically when the message is created using
     <citerefentry><refentrytitle>sd_bus_set_message_new_signal</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
     <citerefentry><refentrytitle>sd_bus_set_message_new_method_call</refentrytitle><manvolnum>3</manvolnum></citerefentry>,
     <citerefentry><refentrytitle>sd_bus_set_message_new_method_error</refentrytitle><manvolnum>3</manvolnum></citerefentry>
-    and similar functions.
+    and similar functions.</para>
+
+    <para><function>sd_bus_message_get_error()</function> returns the error stored in the message
+    <parameter>m</parameter>, if there is any. Otherwise, it returns <constant>NULL</constant>.
+    <function>sd_bus_message_get_errno()</function> returns the error stored in the message
+    <parameter>m</parameter> as a (positive) value from errno, if there is any. Otherwise, it returns zero.
```

Comment:
"from errno" — I don't understand this phase. "as errno-style positive number" ?